defaults/main.yml

```diff
@@ -1,10 +1,13 @@
+mysql_hardening_enabled: no
```

Comment:
We should set this to "yes", because my expectation is that a role just works when I include it into my playbook. Also I've never seen another role you have to "enable" with an additional variable.